member/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Member(models.Model):
+    user = models.OneToOneField(User)
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+
+    def __str__(self):
+        return self.user.get_username()
+
+    def __unicode__(self):
+        return self.user.get_username()
```

Comment:
I used ``python-social-auth`` with facebook and without writing my own models, and ``{{ request.user.username }}`` gives me the user's facebook name, and that's all I know :P